# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rsynth/rsynth-use-devfsdsp.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./config/linuxplay.c.orig	2003-07-25 11:47:04.710496048 +0200
+++ ./config/linuxplay.c	2003-07-25 11:29:16.000000000 +0200
@@ -38,7 +38,7 @@
 
 static int dev_fd = -1;
  /* file descriptor for audio device */
-char *dev_file = "/dev/sbdsp";
+char *dev_file = "/dev/sound/dsp";
 
 static int linear_fd = -1;
 
